main_files/main.py

```diff
@@ -218,6 +220,7 @@ def run_mcor(input_dict):
     # Other SolarProfileGenerator inputs dictionary
     input_dict["other_SolarProfileGenerator_inputs"] = {}
     input_dict["other_SolarProfileGenerator_inputs"]["suppress_warnings"] = False
+    input_dict["other_SolarProfileGenerator_inputs"]["save_timeseries_json"] = False
```

Comment:
This doesn't really have anything to do with the SolarProfileGenerator, so maybe it makes sense to create a new category, maybe "output_inputs" for this parameter. I also noticed that the "save_filename" parameter would maybe sense to be in "output_inputs" rather than "load_inputs" if you don't mind updating that at the same time.